repose-aggregator/components/filters/header-translation-filter/src/test/scala/org/openrepose/filters/headertranslation/HeaderTranslationFilterTest.scala

```diff
@@ -109,40 +141,25 @@ class HeaderTranslationFilterTest extends FunSpec with BeforeAndAfter with Match
 
       filter.doFilter(request, null, mockFilterChain)
 
-      val requestCaptor = ArgumentCaptor.forClass(classOf[HttpServletRequest])
-      verify(mockFilterChain).doFilter(requestCaptor.capture(), any[ServletResponse])
-
-      val capturedRequest = requestCaptor.getValue
-      capturedRequest.getHeaderNames.toSeq should contain theSameElementsAs request.getHeaderNames.toSeq
+      getCapturedRequest.getHeaderNames.toSeq should contain theSameElementsAs request.getHeaderNames.toSeq
     }
 
     it("should remove the original header") {
       filter.doFilter(mockRequest, null, mockFilterChain)
 
-      val requestCaptor = ArgumentCaptor.forClass(classOf[HttpServletRequest])
-      verify(mockFilterChain).doFilter(requestCaptor.capture(), any[ServletResponse])
-
-      val capturedRequest = requestCaptor.getValue
-      capturedRequest.getHeader("X-One") shouldBe null
+      getCapturedRequest.getHeader("X-One") shouldBe null
     }
 
     it("should add a new header with a single value") {
       filter.doFilter(mockRequest, null, mockFilterChain)
 
-      val requestCaptor = ArgumentCaptor.forClass(classOf[HttpServletRequest])
-      verify(mockFilterChain).doFilter(requestCaptor.capture(), any[ServletResponse])
-
-      val capturedRequest = requestCaptor.getValue
-      capturedRequest.getHeaders("X-New-One-One").toSeq should contain theSameElementsAs Seq("valueOne")
+      getCapturedRequest.getHeaders("X-New-One-One").toSeq should contain theSameElementsAs Seq("valueOne")
```

Comment:
I was able to get rid of the "theSameElementsAs" and Seq.apply, but not the .toSeq.